pkg/standalone/bundle_test.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2021 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package standalone
+
+import (
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestParseDetails(t *testing.T) {
+	correctDetails := `{
+		"daprd" : "1.7.0-rc.2",
+		"dashboard": "0.10.0-rc.2",
+		"cli": "1.7.0-rc.2",
+		"daprBinarySubDir": "dist",
+		"dockerImageSubDir": "docker",
+		"daprImageName": "daprio/dapr:1.7.2-rc.2",
+		"daprImageFileName": "daprio-dapr-1.7.2-rc.2.tar.gz"
+	}`
+	f, err := os.CreateTemp("", "*-details.json")
```

Comment:
Just curious why instead of creating this on the fly, we not keeping as a file in folder

Reply:
because we are using ioutil.ReadFile(filename) which actually reads a given file location. That is what is done by the parseDetails function ... It reads a file and parses details from the file ... 
probably renaming the function to `readAndParseDetails` would be better?